src/pycram/ontology/ontology.py

```diff
@@ -813,3 +874,60 @@ def reason(self, world: OntologyWorld = None, use_pellet_reasoner: bool = True)
             return False
         rospy.loginfo(f"{reasoner_name} reasoning finishes!")
         return True
+
```

Comment:
Please also provide tests for such methods. Furthermore stick to the RST docstrings.